config/browserlists.js

```diff
@@ -6,6 +7,7 @@ const allSupportedBrowsers = [
   'ie 11',
 ];
 
+// Browser that can adapt and running
```

Comment:
This isn't quite right, this is the set of browsers we use for the prebuilt assets (though the application can configure whichever ones it wants).